ibis/backends/flink/__init__.py

```diff
@@ -145,6 +145,34 @@ def list_tables(
         # but executing the SQL string directly yields a `TableResult` object
         return self._filter_with_like(tables, like)
 
+    def list_views(
+        self,
+        like: str | None = None,
+        temporary: bool = True,
+    ) -> list[str]:
+        """Return the list of view names.
```

Comment:
the convention elsewhere in Ibis is to fold in views to the output from `list_tables` -- does this need to be a separate API for flink?

Reply:
it's the correct behavior right now that list_table returns views. Good point though this method is only used internally so can be private. I'll make the change.